src/ui/listview.rs

```diff
@@ -314,6 +315,27 @@ impl<I: ListItem + Clone> ViewExt for ListView<I> {
             }
         }
 
+        if cmd == "share" {
+            let source = args.get(0);
+            let url =
+                source.and_then(|source| match source.as_str() {
+                    "selected" => self.content.read().ok().and_then(|content| {
+                        content.get(self.selected).and_then(ListItem::share_url)
+                    }),
+                    "current" => self.queue.get_current().and_then(|t| t.share_url()),
+                    _ => None,
+                });
+
+            return if let Some(url) = url {
+                ClipboardProvider::new()
+                    .and_then(|mut ctx: ClipboardContext| ctx.set_contents(url))
+                    .ok();
+                Ok(CommandResult::Consumed(None))
+            } else {
+                Ok(CommandResult::Ignored)
```

Comment:
We should probably still consume in this case. Otherwise it will display an "unknown command" error, when "share current" is called with no track playing.